e2e-tests/gatsby-script/cypress/integration/scripts-with-sources.ts

```diff
@@ -0,0 +1,174 @@
+import { Script, scripts } from "../../scripts"
```

Comment:
Off main thread tests are blocked because Cypress puts our site in an iframe and it messes with the proxy. Haven't had time to try and fiddle with it further